build.gradle.kts

```
@@ -5,6 +5,22 @@ plugins {
 	id("io.spring.dependency-management") version "1.1.0"
 	kotlin("jvm") version "1.7.21"
 	kotlin("plugin.spring") version "1.7.21"
+
+	/**
+	 * detekt
+	 *
+	 * URL
+	 * - https://github.com/detekt/detekt
+	 * GradlePlugins(plugins.gradle.org)
+	 * - https://plugins.gradle.org/plugin/io.gitlab.arturbosch.detekt
+	 * Main用途
+	 * - Linter/Formatter
+	 * Sub用途
+	 * - 無し
+	 * 概要
+	 * KotlinのLinter/Formatter
+	 */
+	id("io.gitlab.arturbosch.detekt") version "1.2.0"
```

[must]

```suggestion
	id("io.gitlab.arturbosch.detekt") version "1.22.0"
```

Before: 1.2.0
After: 1.22.0

ですかねー？